app/models/concerns/with_assignments.rb

```diff
@@ -18,15 +18,15 @@ def has_messages_for?(user)
     messages_for(user).present?
   end
 
-  def find_assignment_for(user)
+  def find_assignment_for(user, organization)
```

Comment:
:see_no_evil: a dangling `organization` is still here

Reply:
This is on purpose, right?

If that is the case, please put a comment about it, and name it `_organization`

Reply:
Yes, it's all ready to filter by organization by changing
`assignments.find_by(submitter: user)`
to
`assignments.find_by(submitter: user, organization: organization)`
when [the time comes](https://github.com/mumuki/ikumi-online/issues/464).

Reply:
I'll change it to `_organization`, didn't know about the convention